<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KDE CDEmu Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeWidget" name="deviceList">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string>Device</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Image</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="buttonWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="addDevice">
         <property name="text">
          <string>Add Device</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeDevice">
         <property name="text">
          <string>Remove Device</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>His&amp;tory</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Setti&amp;ngs</string>
    </property>
    <addaction name="actionTrayIcon"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHistory"/>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionTrayIcon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show in System Tray</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>showFileSelector()</slot>
 </slots>
</ui>
